assets/css/detours/_detour_map.scss

```diff
@@ -31,11 +31,16 @@
 }
 
 .c-detour_map-circle-marker--start {
-  stroke: $color-kiwi-400;
-  fill: $color-kiwi-400;
+  stroke: #03a900;
+  fill: #03a900;
```

Comment:
**suggestion (non-blocking):** I wonder if it would be possible to find a home for this in `_tokens_2024.scss`. Perhaps it could be sort of like `$missed-stop`, except it doesn't need to be added to the Bootstrap theme colors. But, this is probably a genuine one-off so if there's no good place for it I wouldn't sweat it.

Reply:
[how does this look?](https://github.com/mbta/skate/pull/2468/commits/253edeb35ff96c08ad05317757be14579a0086ab)

I agree that this is a good change, since things that are one-offs often don't stay one-offs 😅 I just didn't know the best way to do it!

Reply:
Yeah, I'm not sure this is the best long-term solution if that list grows but I think for now it's a good starting point.